Anlab.Mvc/Controllers/LabController.cs

```diff
@@ -25,6 +26,8 @@ public class LabController : ApplicationController
         private readonly IOrderMessageService _orderMessageService;
         private readonly IFileStorageService _fileStorageService;
 
+        private readonly int _maxShownOrders = 1000;

```

Comment:
@laholstege since it is statically set, make it a const or a static readonly.  Const probably makes the most sense